src/tracker/ErrorBar/index.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React from 'react'
+import { connect } from 'react-redux'
+import { compose, withHandlers } from 'recompose'
+
+import Snackbar from '@material-ui/core/Snackbar'
+import IconButton from '@material-ui/core/IconButton'
+import CloseIcon from '@material-ui/icons/Close'
+
+import { withStyles } from '@material-ui/core/styles'
+
+import { setErrorMessage as _setErrorMessage } from '../../actions/utilActions'
+
+const styles = theme => ({
+  errorMessage: {
+    bottom: '120px'
+  }
+})
+
+type Props = {|
+  errorMessage: ?string,
+  classes: Object,
+  handleClose: () => void
+|}
+
+const ErrorBar = ({ errorMessage, handleClose, classes }: Props) => (
+  <Snackbar
+    anchorOrigin={{ vertical: 'bottom', horizontal: 'left' }}
```

Comment:
write this into the styles constant and write here something like:
```suggestion
    anchorOrigin={classes.anchorOrigin}
```

Reply:
Is this actually doable? As I see it, styles are used to modify css and parts of theme. anchorOrigin is a prop.

Reply:
hm, depends on how the anchorOrigin works. maybe it can accept also a class name with the styles. please try and if it doesn't work, just resolve my comment :)